proposals/0024-opacity-micromaps.md

```diff
@@ -253,32 +236,22 @@ specified.
 Three DXIL operations accept `RayFlags` as input, but only one requires this
 input to be immediate: `AllocateRayQuery`.
```

Comment:
This isn't accurate anymore.  There are two trace ops and two allocate ops (for which this must be immediate) now.